docs/validator/keys.md

```diff
@@ -0,0 +1,63 @@
+
+**Validator keys** are used by validators (people and companies who are committed to maintaining the integrity of the system) to support their work of validating blocks and chunks on the network, nothing more. The human validators don't think about these keys beyond creating them and resetting them. Once added to a validator's node, validator keys are used by the node to do their thing of validating blocks and chunks.  As a convenience to validators, validator keys are currently produced by a script at node startup if they don't already exist (in the case of NEAR Stake Wars, the `start_stakewars.py` script) but this may change.
+
+**Node keys** are something no humans on the network think about except core contributors to the platform.  These keys are used internally by a node to sign low-level communications with other nodes in the network like sending block headers or making other verifiable requests.  Node keys are currently provided to a node at startup by a script.  In the case of NEAR Stake Wars it's the `start_stakewars.py` script that produces these keys for now, but this may change.
+
+## can keys be changed?
+
+Yes, but only in that keys can be *reset* (ie. regenerated as a new key pair). If a private key is lost or compromised somehow then a new key pair must be generated.  This is just the nature of secure keys.
+
+**Signers** can create new keys and revoke existing keys at will. NEAR Wallet also supports key recovery via SMS or seed phrase which makes it convenient to move signer keys from one computer to another, for example.
+
+**Validators** have the option to reset their validator keys at any time but it makes sense to avoid resetting validator keys while staking.  To reset their keys, a human validator stops their node, changes their validator key and restarts the node.  All new validator output will be signed by these new keys.
+
+**Nodes** should not need to reset their node keys keys
+
+<blockquote class="info">
+<strong>did you know?</strong><br><br>
+
+As a brief word on the NEAR runtime, the subsystem that manages state transitions on the blockchain (ie. keeping things moving from one block to the next), it's worth understanding that the movement of the system happens in stages, called epochs, during which the group of validators does not change. The [Nightshade whitepaper](http://near.ai/nightshade) introduces epochs this way: "the maintenance of the network is done in epochs, where an epoch is a period of time on the order of days."  and there's much more detail in the paper.
```

Comment:
I thought that Alex mentioned an epoch would be half a day? @SkidanovAlex

Reply:
Yes that's the plan. Although half a day is still on the order of days I guess